src/Data/toClientParams.ts

```diff
@@ -0,0 +1,41 @@
+import { provideDataClass } from 'scrivito'
+
+type DataConnection = Parameters<typeof provideDataClass>[1]['connection']
+type IndexCallback = NonNullable<DataConnection['index']>
+type IndexParams = Parameters<IndexCallback>[0]
+
+interface OperatorSpec {
+  operator: 'equals' | 'notEquals'
+  opCode: 'eq' | 'neq'
+  value: string
+}
+
+export function toClientParams(params: IndexParams) {
+  return {
+    ...toClientFilterParam(params.filters()),
+    _continuation: params.continuation(),
+    _order: params.order().length
+      ? params
+          .order()
+          .map((o) => o.join('.'))
+          .join(',')
+      : undefined,
+    _limit: params.limit().toString(),
+    _search: params.search() || undefined,
```

Comment:
* [ ] FYI Not sure about the `|| undefined`. Looks like a safeguard against accidental empty-string searches, but don't we hide potential application errors with this?

Reply:
This code is inspired by the SDK.

Reply:
> This code is inspired by the SDK.

I don't understand. If the SDK would autoconvert `''` to `undefined` we wouldn't need this the fallback here at all.

Reply:
It's inspired by `js/src/scrivito_sdk/data_integration/create_standard_api_connection.ts#L109`. This is actually the behavior of `provideDataClass` with `restApi`.